src/plugins/hexnumber/README.md

```diff
@@ -0,0 +1,31 @@
+- infos = Information about the template plugin is in keys below
```

Comment:
Please replace ”template“ with the name of the plugin (“hexnumber“).

Reply:
Thank you!

The `../../scripts/copy-template` approach avoids such problems.